tests/integration/routes/test_view_submitted_response_pdf.py

```diff
@@ -0,0 +1,66 @@
+from time import sleep
+
+from app import settings
+from tests.integration.routes.test_view_submitted_response import (
+    ViewSubmittedResponseBase,
+)
+
+
+class TestViewSubmissionResponsePDF(ViewSubmittedResponseBase):
+    def test_download_when_submitted_response_not_enabled(self):
+        # Given I launch and complete a questionnaire that does not have view-submitted-response enabled
+        self.launchSurvey("test_confirmation_email")
+        self.post()
+        self.post()
+
+        # When I try to download my answer
+        self.get("/submitted/download-pdf")
+
+        # Then a 404 page is returned
+        self.assertStatusNotFound()
+
+    def test_download_when_submitted_response_enabled_but_not_expired(self):
+        # Given I launch and complete a questionnaire that does not have view-submitted-response enabled
+        self._launch_and_complete_questionnaire()
+
+        # When I try to download my answer
+        self.get(self.VIEW_RESPONSE_PAGE_URL)
+        self.get(self.get_download_button().attrs["href"])
+
+        # Then a the download is successful
+        self.assertStatusOK()
+
+        # Check response content type is PDF
+        self.assertEqual(self.last_response.content_type, "application/pdf")
+
+        # Check file is set to download and not open inline
+        assert "attachment;" in self.last_response_headers["Content-Disposition"]
+
+        # Check filename is set as expected
+        assert (
+            "filename=test_view_submitted_response.pdf"
+            in self.last_response_headers["Content-Disposition"]
+        )
+
+        # Check content length is reasonable.
+        # This is given some leeway as it can change with DS changes.
+        self.assertGreater(self.last_response.content_length, 16500)
```

Comment:
why not `assert self.last_response.content_length > 16500 ,"PDF content is less than 16500"`